### PR TITLE
Change oversized buttons

### LINKED ISSUE #12354: [frio] Buttons in comment form are oversized
- [ o ] I have searched open and closed issues for duplicates

I notice that now buttons on new comment form became oversized. It is not much problem on desktop but on mobile they take a lot of space.

Desktop example:
image

Mobile example:
image

I use Firefox but also tried Edge.

### PR DESCRIPTION
**Pull request by loma-one**
Additional css specification in global.css to resize the button in the composer. ".template-icon" Use ".template-icon" in additional buttons that are currently displayed oversized.

### CODE REVIEW

**Comment by loma-one** [comment]
https://github.com/friendica/friendica/pull/12370

File: view/templates/item/compose.tpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="generic-page-wrapper">
-	<h2>{{$l10n.compose_title}}</h2>
+	<h4>{{$l10n.compose_title}}</h4>
```
Comment (MrPetovan) [inline view/templates/item/compose.tpl]: Please don't change the tag name to change the style.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by MrPetovan** [comment]
Depends on #12370

**Comment by loma-one** [comment]
Where do I have to look for the comment field to resize the icon there as well. I am not familiar with the template structure unfortunately and am trying to find my way around. Thanks a lot

**Comment by MrPetovan** [comment]
They are defined there: [view/theme/frio/templates/comment_item.tpl](https://github.com/friendica/friendica/blob/2022.12-rc/view/theme/frio/templates/comment_item.tpl)

**Review by loma-one — COMMENTED** [review]
Please adopt with the change H2 for the title

**Comment by MrPetovan** [comment]
Do you know you can add your changes to an already existing branch like the one in this PR named `patch-1`? 🙂

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**